composer.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "ezsystems/behatbundle",
+    "name": "ibexa/behat",
```

Comment:
I think there is missing `replace` section here.

Reply:
Added in https://github.com/ibexa/behat/pull/1/commits/b028cd15bcb6f413e19b482dce9b1fa133b92384